compiler/damlc/daml-preprocessor/src/DA/Daml/Preprocessor.hs

```diff
@@ -163,8 +170,8 @@ checkModuleName (GHC.L _ m)
 -- | We ban people from importing modules such
 checkImports :: GHC.ParsedSource -> [(GHC.SrcSpan, String)]
 checkImports x =
-    [ (ss, "Import of internal module " ++ GHC.moduleNameString m ++ " is not allowed.")
-    | GHC.L ss GHC.ImportDecl{ideclName=GHC.L _ m} <- GHC.hsmodImports $ GHC.unLoc x, isInternal m]
+    [ (ss, "Import of unstable internal module " ++ GHC.moduleNameString m ++ " is not allowed.")
```

Comment:
```suggestion
    [ (ss, "Import of internal module " ++ GHC.moduleNameString m ++ " is not allowed.")
```
I’d just keep it and say that the other ones are not internal even if they might have `Internal` in the name for historic reasons.

Reply:
Done